### ISSUE TITLE
The difference of input between pifu and pifuhd

### ISSUE THREAD

**Issue by SFM2020** [facebookresearch/pifuhd#38]
why pifuhd dont need mask but pifu highly dependent on mask ?

**Comment by shunsukesaito** [comment]
Good question! After all, it's a design choice. I didn't like having external module for segmentation in PIFu and decided to make PIFuHD more robust to random background.

**Comment by SFM2020** [comment]
I wanna change pifu’s input the same as pifuhd ，how should I do?

**Comment by shunsukesaito** [comment]
You need to train a PIFu model with random backgrounds. This requires retraining the model by yourself.

**Comment by SFM2020** [comment]
I have retrained pifu with mask as input ,if i wanna remove mask，should I change the code like pifuhd as below?
` # if the point is outside bounding box, return outside.
        in_bb = (xyz >= -1) & (xyz <= 1)
        in_bb = in_bb[:, 0, :] & in_bb[:, 1, :] & in_bb[:, 2, :]
        in_bb = in_bb[:, None, :].detach().float()
        if labels is not None:
            self.labels = in_bb * labels`
